flink-core/pom.xml

```diff
@@ -33,6 +33,25 @@ under the License.
 
 	<packaging>jar</packaging>
 
+	<properties>
+		<surefire.module.config> <!--
+			required by JmxServer
+			-->--add-exports=java.rmi/sun.rmi.registry=ALL-UNNAMED <!--
+			PluginConfigTest
+			-->--add-opens=java.base/java.util=ALL-UNNAMED <!--
+			ExceptionUtilsTest
+			-->--add-opens=java.base/java.lang=ALL-UNNAMED <!--
+			-->--add-opens=java.base/java.lang.invoke=ALL-UNNAMED <!--
+			StateDescriptorTest
+			-->--add-opens=java.base/java.io=ALL-UNNAMED <!--
+			SerializersTest
+			-->--add-opens=java.base/java.net=ALL-UNNAMED <!--
+			InitOutputPathTest
+			-->--add-opens=java.base/java.util.concurrent.locks=ALL-UNNAMED <!--
```

Comment:
wouldn't that be rather a `--add-exports` because we're using it only for extending the class? :thinking:

Reply:
TBH I let the JVM error messages guide me.

We need this because we change the accessibility of the `ReentrantLock` in the `FileSystem`.